main.go

```diff
@@ -88,6 +97,28 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Get STS vars
+	// ROLEARN env var is set via operator subscription
+	roleARN := os.Getenv("ROLEARN")
```

Comment:
should we check if ENV VAR is set or it will be always present?

Reply:
Done, updated